src/containers/LearningResourcePage/LearningResourcePage.jsx

```diff
@@ -29,21 +29,23 @@ class LearningResourcePage extends Component {
   }
 
   render() {
-    const { locale, match, history, isSaving, licenses } = this.props;
-
+    const {
+      locale,
+      match,
+      history,
+      isSaving,
+      licenses,
+      showSaved,
+    } = this.props;
+    const sendDown = { isSaving, licenses, locale, showSaved };
```

Comment:
Kan vi ikke bare bruke `...rest`?

Reply:
Gjorde det for å ikke sende ned props man ikke trenger som fetchLicense, men det gjør ikke meg noe. Endret det til ...rest